tests/scripts/framework/run_test.sh

```diff
@@ -98,6 +100,12 @@ do
         --run=*)
             RUN_TEST=${i#*=}
             ;;
+        --configdir=*)
+            TEST_ROOT_PATH=${i#*=}
```

Comment:
Is the config directory the same as `TEST_ROOT_PATH`?

Reply:
>Is the config directory the same as `TEST_ROOT_PATH`?

Yes.